src/platform/tls_mitls.c

```
@@ -1344,6 +1333,41 @@ QuicTlsProcessDataComplete(
                     State->SessionResumed = TRUE;
                     TlsContext->EarlyDataAttempted = TRUE;
                     State->EarlyDataState = QUIC_TLS_EARLY_DATA_ACCEPTED;
+                    //
+                    // Get resumption data from the client hello
+                    //
+                    uint32_t PreviousOffset = BufferOffset - (uint32_t)Context.consumed_bytes;
+                    mitls_hello_summary HelloSummary = { 0 };
+                    uint8_t* Cookie = NULL;
+                    size_t CookieLen = 0;
+                    uint8_t* Ticket = NULL;
+                    size_t TicketLen = 0;
+                    if (!FFI_mitls_get_hello_summary(
+                            TlsContext->Buffer + PreviousOffset, TlsContext->BufferLength - PreviousOffset,
+                            FALSE, // TODO: Not sure if this is the correct value...
+                            &HelloSummary,
+                            &Cookie, &CookieLen,
+                            &Ticket, &TicketLen)) {
+                        QuicTraceLogConnError(
+                            miTlsFfiProcessFailed,
+                            TlsContext->Connection,
+                            "FFI_mitls_get_hello_summary failed, cookie_len: %zu, ticket_len: %zu",
+                            CookieLen,
+                            TicketLen);
+                        ResultFlags |= QUIC_TLS_RESULT_ERROR;
```

Bail on failure instead of continuing?